.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a bug report
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+If you encounter an issue, please attempt the following steps :
+* check our [public documentation](https://docs.prophesee.ai/stable/index.html)
+* make sure you are using the last version of OpenEB
+
+If the earlier steps didn't solve the issue, raise an issue providing:
+* information about your system with [Metavision Platform Info Application](https://docs.prophesee.ai/stable/metavision_sdk/modules/metavision_hal/samples/platform_info.html)
+* your operating system and its version
+* your camera type and its serial number that can be found on your camera's base (bottom) plate
+* software that has an issue, expected and actual behavior, and as detailed as possible reproduction steps 
+
```

Comment:
Should we add an item about sharing of RAW recordings, to enable reproducing the results on our side?